server/util/cosmos.js

```diff
@@ -20,7 +20,7 @@ import { PUBSUB_TOPIC_MISC } from '../constant';
 import publisher from './gcloudPub';
 
 import {
-  COSMOS_ENDPOINT as cosmosRPCEndpoint,
+  COSMOS_RPC_ENDPOINT as cosmosRPCEndpoint,
```

Comment:
```suggestion
  COSMOS_RPC_ENDPOINT,
```
Prefer to skip the rename step and just use `COSMOS_RPC_ENDPOINT` as the parameter like the rest of the config parameters.